src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -147,6 +147,14 @@ final case class RequestSession[F[_]: Monad: Trace](
       .send(sttpBackend)
       .map(_.body)
 
+  def head(
+      uri: Uri
```

Comment:
might also make sense to specify `accept` in case response can vary depending on it. `get` also has parameter for `contentType` but that one we probably don't need